<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>381</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="carBrand">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>360</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="carModel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>400</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="carYearOfManufacture">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>440</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="checkdatabasebutton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Check cars</string>
    </property>
   </widget>
   <widget class="QPushButton" name="additemsbutton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>270</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add car </string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteitembutton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>330</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete car</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editdatabasebutton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit car</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="carId">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>520</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textCars">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>411</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="checkownerdatabutton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Check owners</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addownerdatabasebutton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>270</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add owner</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteownerdatabasebutton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>330</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete owner</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editownerdatabasebutton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>300</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit owner</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ownerId">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>520</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ownerFirstname">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>360</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ownerAddress">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>440</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ownerLastname">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>400</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textOwners">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>411</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textCarOwners">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>453</y>
      <width>411</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="checkcarsownerships">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>570</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Check cars ownerships</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>490</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Täytä ID vain jos muokkaat tai poistat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addownershipsbutton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>600</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add ownerships</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteownershipsbutton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>600</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete ownerships</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editownershipsbutton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>600</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit ownerships</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
